modules/openxBidAdapter.js

```diff
@@ -181,8 +181,8 @@ function buildCommonQueryParamsFromBids(bids) {
   const isInIframe = utils.inIframe();
 
   return {
-    ju: config.getConfig('pageUrl') || utils.getTopWindowUrl(),
-    jr: utils.getTopWindowReferrer(),
+    ju: bids[0].params.referrer || config.getConfig('pageUrl') || utils.getTopWindowUrl(),
```

Comment:
what if the bids[0] doesn't exist?

Reply:
they are using it at https://github.com/Marfeel/Prebid.js/blob/7ea8060bcb89f82ae297da512147eba7955b35a7/modules/openxBidAdapter.js#L202

i think it is pretty safe